.github/workflows/integration-tests.yml

```diff
@@ -12,14 +12,14 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
What happens when 23 is out? Don't we wish to have more granularity and control over the version?

Reply:
this small CI isn't that complex, I'm o.k. with using latest.
rather then get it break cause of deprecations